AppSandboxFileAccess/Classes/AppSandboxFileAccess.m

```diff
@@ -181,15 +187,23 @@ - (BOOL)requestAccessPermissionsForFileURL:(NSURL *)fileURL persistPermission:(B
 	fileURL = [[fileURL URLByStandardizingPath] URLByResolvingSymlinksInPath];
 	
 	// lookup bookmark data for this url, this will automatically load bookmark data for a parent path if we have it
-	NSData *bookmarkData = [AppSandboxFileAccessPersist bookmarkDataForURL:fileURL];
+	NSData *bookmarkData = (self.bookmarkPersistanceDelegate) ? [self.bookmarkPersistanceDelegate bookmarkDataForURL:fileURL] : [AppSandboxFileAccessPersist bookmarkDataForURL:fileURL];
+	
 	if (bookmarkData) {
 		// resolve the bookmark data into an NSURL object that will allow us to use the file
 		BOOL bookmarkDataIsStale;
-		allowedURL = [NSURL URLByResolvingBookmarkData:bookmarkData options:NSURLBookmarkResolutionWithSecurityScope|NSURLBookmarkResolutionWithoutUI relativeToURL:nil bookmarkDataIsStale:&bookmarkDataIsStale error:NULL];
+		allowedURL = [NSURL URLByResolvingBookmarkData:bookmarkData
+											   options:NSURLBookmarkResolutionWithSecurityScope|NSURLBookmarkResolutionWithoutUI
+										 relativeToURL:nil
+								   bookmarkDataIsStale:&bookmarkDataIsStale
+												 error:NULL];
```

Comment:
This line doesn't look like it changed other than formatting. Could this line be reverted.